src/command.ls

```diff
@@ -246,6 +250,8 @@ function output-filename filename, json
 # - __^C__: Cancel input if any. Quit otherwise.
 # - __??__: <https://github.com/joyent/node/blob/master/lib/readline.js>
 !function repl
+  MAX-HISTORY-SIZE = 500
```

Comment:
maybe we want to move that up and use `const` just to signify it's a constant. LGTM otherwise